utils/trixi2txt.jl

```diff
@@ -294,7 +294,7 @@ function unstructured2structured(unstructured_data::AbstractArray{Float64},
       # Interpolate data
       vandermonde = vandermonde_per_level[level + 1]
       @views structured[first:last, v] .= (
-           reshape(multiply_dimensionwise(reshaped_data[:, :, element_id], vandermonde),
+           reshape(multiply_dimensionwise_naive(reshaped_data[:, :, element_id], vandermonde),
```

Comment:
To be honest, there was an error in Tullio I did not understand and I *assumed* it is related to passing a reshaped array. Thus the easiest fix for me (to get @martin-pmf and @FelipeSantillan up and running) was this. Maybe you can make something of this error message, otherwise I'd suggest to leave it as it is and have it fixed by obsolescence (Felipe is working on 1D plotting as of this week):

Prior to the fix, I got the following error (`julia --project=.`):
```julia
julia> using Revise, Trixi

julia> trixi_include("examples/1d/elixir_advection_basic.jl")

julia> includet("utils/trixi2txt.jl")

julia> using .Trixi2Txt

julia> Trixi2Txt.trixi2txt("out/solution_000000.h5")
ERROR: MethodError: no method matching stridedpointer(::Ptr{Float64}, ::ArrayInterface.Contiguous{1}, ::Nothing, ::Nothing, ::Tuple{ArrayInterface.StaticInt{8},Int64}, ::Tuple{ArrayInterface.StaticInt{1},ArrayInterface.StaticInt{1}})
Closest candidates are:
  stridedpointer(::Ptr{T}, ::ArrayInterface.Contiguous{C}, ::ArrayInterface.ContiguousBatch{B}, ::ArrayInterface.StrideRank{R}, ::X, ::O) where {T<:Union{Bool, Float32, Float64, Int16, Int32, Int64, Int8, UInt16, UInt32, UInt64, UInt8, VectorizationBase.Bit}, C, B, R, N, X<:Tuple{Vararg{Any,N}}, O<:Tuple{Vararg{Any,N}}} at /home/mschlott/.julia/packages/VectorizationBase/AzQKz/src/strided_pointers/stridedpointers.jl:74
Stacktrace:
 [1] stridedpointer at /home/mschlott/.julia/packages/VectorizationBase/AzQKz/src/strided_pointers/stridedpointers.jl:72 [inlined]
 [2] 𝒜𝒸𝓉! at /home/mschlott/.julia/packages/Tullio/u3myB/src/macro.jl:1090 [inlined]
 [3] 𝒜𝒸𝓉! at /home/mschlott/.julia/packages/Tullio/u3myB/src/macro.jl:1087 [inlined]
 [4] threader at /home/mschlott/.julia/packages/Tullio/u3myB/src/threads.jl:48 [inlined]
 [5] macro expansion at /home/mschlott/.julia/packages/Tullio/u3myB/src/macro.jl:1000 [inlined]
 [6] multiply_dimensionwise!(::Array{Float64,2}, ::SubArray{Float64,2,Base.ReshapedArray{Float64,3,SubArray{Float64,2,Array{Float64,3},Tuple{Base.Slice{Base.OneTo{Int64}},Base.Slice{Base.OneTo{Int64}},Int64},true},Tuple{}},Tuple{Base.Slice{Base.OneTo{Int64}},Base.Slice{Base.OneTo{Int64}},Int64},true}, ::Array{Float64,2}) at /mnt/ssd/home/mschlott/hackathon/plot-Trixi.jl/src/solvers/dg/interpolation.jl:107
 [7] multiply_dimensionwise(::SubArray{Float64,2,Base.ReshapedArray{Float64,3,SubArray{Float64,2,Array{Float64,3},Tuple{Base.Slice{Base.OneTo{Int64}},Base.Slice{Base.OneTo{Int64}},Int64},true},Tuple{}},Tuple{Base.Slice{Base.OneTo{Int64}},Base.Slice{Base.OneTo{Int64}},Int64},true}, ::Array{Float64,2}) at /mnt/ssd/home/mschlott/hackathon/plot-Trixi.jl/src/solvers/dg/interpolation.jl:70
 [8] unstructured2structured(::Array{Float64,3}, ::Array{Int64,1}, ::Int64, ::Array{Int64,1}) at /mnt/ssd/home/mschlott/hackathon/plot-Trixi.jl/utils/trixi2txt.jl:296
 [9] trixi2txt(::String; variables::Array{Any,1}, output_directory::String, nvisnodes::Nothing, max_supported_level::Int64) at /mnt/ssd/home/mschlott/hackathon/plot-Trixi.jl/utils/trixi2txt.jl:85
 [10] trixi2txt(::String) at /mnt/ssd/home/mschlott/hackathon/plot-Trixi.jl/utils/trixi2txt.jl:25
 [11] top-level scope at REPL[6]:1

julia> 


```

Reply:
Okay, we can leave it as it is right now :+1: